src/Symfony/Component/Notifier/Notification/Notification.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Notification;
+
+use Psr\Log\LogLevel;
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\Notifier\Receiver\Receiver;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class Notification
+{
+    private static $levels = [
+        LogLevel::DEBUG => 100,
+        LogLevel::INFO => 200,
+        LogLevel::NOTICE => 250,
+        LogLevel::WARNING => 300,
+        LogLevel::ERROR => 400,
+        LogLevel::CRITICAL => 500,
+        LogLevel::ALERT => 550,
+        LogLevel::EMERGENCY => 600,
+    ];
+
+    public const IMPORTANCE_URGENT = 'urgent';
+    public const IMPORTANCE_HIGH = 'high';
+    public const IMPORTANCE_MEDIUM = 'medium';
+    public const IMPORTANCE_LOW = 'low';
+
+    private $channels = [];
+    private $subject = '';
```

Comment:
The assigment can be safely removed as the constructor overrides it, same for `$channels` property

Reply:
done